README.md

```diff
@@ -116,7 +116,7 @@ Because the subcommands and flags are constrained to benefit rapid plugin protot
 - `CADDY_VERSION` sets the version of Caddy to build.
 - `XCADDY_RACE_DETECTOR=1` enables the Go race detector in the build.
 - `XCADDY_SKIP_CLEANUP=1` causes xcaddy to leave build artifacts on disk after exiting.
-
+- `XCADDY_NO_BUILD=1` causes xcaddy to not compile the program, it is used in conjunction with build tools such as [GoReleaser](https://goreleaser.com). Implies `XCADDY_SKIP_CLEANUP=1`.
```

Comment:
Can we change this to `XCADDY_SKIP_BUILD` for consistency with `XCADDY_SKIP_CLEANUP`?

Reply:
@dunglas :point_up: will you still make these changes?

Reply:
Oops, this PR was no longer in my line of sight. Change done!